src/Entities/Entity.h

```diff
@@ -545,6 +549,12 @@ class cEntity
 	/** Returs whether the entity has any mob leashed to */
 	bool HasAnyMobLeashed() const { return m_LeashedMobs.size() > 0; }
 
+	/** a lightweight calculation approach to get explosion exposure rate
+	@param a_ExplosionPosition : explosion position
+	@param a_ExlosionPower : explosion power
+	@return : exposure rate */
```

Comment:
You shouldn't put colon's in the doxy-comments, it will look wrong in the generated docs. i.e. something like
### Returns
        : exposure rate